src/useCases/saveCachedRegistration.ts

```diff
@@ -0,0 +1,13 @@
+import { FormCacheFactory } from "../lib/formCache";
+import { Registration } from "../lib/registration/registration";
+
+export type SaveCachedRegistrationFn = (
+  submissionId: string,
+  registration: Registration
+) => Promise<void>;
+
+export const saveCachedRegistration: SaveCachedRegistrationFn = async (
+  submissionId: string,
+  registration: Registration
+): Promise<void> =>
+  await FormCacheFactory.getCache().set(submissionId, registration);
```

Comment:
This is deliberately not tested as it doesn't contain logic, just a simple delegation to the gateway equivalent.